.pre-commit-config.yaml

```diff
@@ -102,3 +102,4 @@ repos:
     hooks:
       - id: flake8
         name: Check PEP8
+        exclude: docs/source-app
```

Comment:
👍

Reply:
@Borda I don't want formatting on the Lightning App docs. I have optimized the code to be responsive on mobile and black breaks this. We should customize black on docs to be at 80 letter lines.